src/server/views/help/accessibility-statement.html

```diff
@@ -3,14 +3,31 @@
 {% set pageTitle = "Accessibility statement" %}
 
 {% block content %}
-<div class="govuk-width-container">
-    <div class="govuk-main-wrapper ">
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-l">Accessibility Statement</h1>
-                <p class="govuk-body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget volutpat tortor. Duis placerat ornare ipsum, eget molestie arcu tincidunt eget. Nam luctus arcu eu pulvinar mattis. Sed lobortis venenatis quam vel semper. Nunc lacinia augue eu nisl placerat, quis dignissim diam rutrum. Vestibulum sed leo pellentesque, gravida nibh at, consequat purus. Proin ultrices, arcu ac cursus accumsan, tellus mauris tincidunt erat, quis pellentesque sapien tortor sit amet erat.</p>
-            </div>
-        </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Accessibility Statement for ‘{{ serviceName }}’</h1>
+      <p class="govuk-body">This accessibility statement applies to forms created using the Defra Forms Designer.</p>
+      <p class="govuk-body">Defra is committed to making its forms designer accessible, in accordance with the Public
+        Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+
+      <h2 class="govuk-heading-m">Compliance status</h2>
+      <p class="govuk-body">The forms created using the Defra Forms Designer have been tested against the Web Content
+        Accessibility Guidelines (WCAG) 2.2 AA standard.</p>
+      <p class="govuk-body">The forms are fully compliant with the Web Content Accessibility Guidelines version 2.2 AA
+        standard, due to the exemptions listed below.</p>
+
+      <h2 class="govuk-heading-m">Preparation of this accessibility statement</h2>
+      <p class="govuk-body">This statement was prepared on 24 June 2024.</p>
+
+      <h2 class="govuk-heading-m">Feedback and contact information</h2>
+      <p class="govuk-body">If you find any problems not listed on this page or think we’re not meeting accessibility
+        requirements, contact <a class="govuk-link" href="mailto:defraforms@defra.gov.uk">defraforms@defra.gov.uk</a>.</p>
+
+      <h2 class="govuk-heading-m">Enforcement procedure</h2>
+      <p class="govuk-body">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector
+        Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility
+        regulations’). If you’re not happy with how we respond to your complaint, contact the <a class="govuk-link"
```

Comment:
Probably need `rel="external"` and we're done

Reply:
Resolved with 74b4cdf97d194eac9af437d87148c7248b4abeb3